### ISSUE TITLE
Is that DensePose must be installed for deploy in ComfyUI？

### ISSUE THREAD

**Issue by pzc163** [Zheng-Chong/CatVTON#5]
OS:win10
python：3.10
torch:2.1.2
cuda:12.1

when i deploy this node in comyui, it has come below import error,it seems like DensePose is  missing. so does that DensePose have to be installed？

![image](https://github.com/user-attachments/assets/9258d13a-682b-4635-9d17-a4f3738e91f9)

**Comment by Zheng-Chong** [comment]
Yes, since Gradio and ComfyUI App involve the functionality of **automatically generating Masks**, DensePose is required to guide the process, hence the need to additionally install _Detectron2 and DensePose_. We have updated the description in the installation guide. If you only need to perform inference on a dataset, these two packages are not required.

**Comment by pzc163** [comment]
i found that its very hard to install DensePose in Windows OS, could you provide a DensePose wheel file for Windows OS user？

**Comment by Zheng-Chong** [comment]
DensePose is maintained by Facebook's Detectron, and we do not have experimental machines in the Windows environment, making it difficult for us to provide specific Windows installation instructions. However, I have found similar issues encountered by others, and I hope they can be of help to you, refer to this GitHub issue: [vid2densepose #14](https://github.com/Flode-Labs/vid2densepose/issues/14).

**Comment by pzc163** [comment]
thanks for your guideline，I have succeed import catVTON node in Comfyui. So if I need to download any dataset or pre-trained weight for this project now？thx~
![image](https://github.com/user-attachments/assets/b7ca8785-7ac4-46d3-a216-39b505d0c745)

**Comment by Zheng-Chong** [comment]
For ease of use, we have integrated the weight download into the code, and the weights will be automatically downloaded to the local area the first time you run the workflow.
Of course, you can also manually download the weights from HuggingFace and fill in the local paths for `sd15_inpainting` and `catvton` into the **_"Model Loading"_** text box.

**Comment by Zheng-Chong** [comment]
For datasets, if you do not need to evaluate on public datasets (which is usually required for scientific research) and are only using CatVTON for testing and playing, you can directly upload **_any suitable images_** without considering the download of the datasets.

**Comment by pzc163** [comment]
It looks like the clothing change mission failed in two different types of dressup missions 
![image](https://github.com/user-attachments/assets/4ff36923-cc52-4015-9a08-60dfc77f080b)

**Comment by Zheng-Chong** [comment]
The Chinese localization version has overlaid the original input image Titles; 
The input image on the left should be the person, and on the right is the reference clothing.
refer to the following image:

![352720549-ed80dc42-02e4-42a4-be1c-51eee20e9119](https://github.com/user-attachments/assets/938f811a-ed8e-4903-9dd7-6b69d347840f)

**Comment by pzc163** [comment]
thx for your work, it seems like test success now!
![image](https://github.com/user-attachments/assets/3136289b-8d50-4fe4-a800-1712f0265554)

**Comment by Zheng-Chong** [comment]
😁 Have fun with CatVTON !

**Comment by Zheng-Chong** [comment]
This issue is closed and if there is any other question in Windows OS, please come to https://github.com/Zheng-Chong/CatVTON/issues/8 .
